src/objects.c

```diff
@@ -991,7 +991,8 @@ CK_RV p11prov_obj_find(P11PROV_CTX *provctx, P11PROV_SESSION *session,
     P11PROV_debug("Find objects [class=%lu, id-len=%lu, label=%s]", class,
                   id.ulValueLen,
                   label.type == CKA_LABEL ? (char *)label.pValue : "None");
-
+    
+    memset (template, 0, sizeof (template));
```

Comment:
again no, the original initialization is just fine.

Reply:
I won't repeat it again for each occurence, but please revert all of these changes